site/docs/v1/tech/apis/themes.adoc

```diff
@@ -69,6 +70,65 @@ include::docs/v1/tech/apis/_theme-response-body.adoc[]
 include::docs/v1/tech/apis/_theme-response-body-suffix.adoc[]
 {%capture singleResponse %}false{% endcapture %}
 
+== Create a New Theme From an Existing Theme
+
+This API is used to create a new Theme from an existing Theme.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Theme with a randomly generated Id
```

Comment:
```suggestion
link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Theme from an existing Theme with a generated Id
```

Reply:
We use `generated` and `randomly generated` interchangeably.  Is there a preference here?